submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,170 @@
+                                    <img class="popup_img" src="./popup-icons/search.png" alt="search">
+                                    <p>Search</p>
+                                </a>
+                            </li>
+
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/chat.png" alt="chat">
+                                    <p>Chat</p>
+                                </a>
+                            </li>
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/classroom.png" alt="classroom">
+                                    <p>classroom</p>
+                                </a>
+                            </li>
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/collection.png" alt="collection">
+                                    <p>Collection</p>
+                                </a>
+                            </li>
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/documents.png" alt="documents">
+                                    <p>Documents</p>
+                                </a>
+                            </li>
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/mars.png" alt="mars">
+                                    <p>Mars</p>
+                                </a>
+                            </li>
+                            <li class="popup_item popup_more_items">
+                                <a href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/promo.png" alt="promo">
+                                    <p>Promo</p>
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+
+                </li>
+
+                <li class="nav_item">
+                    <a href="#" target="_blank" rel="noopener noreferrer">
+                        <img class="nav-img" src="./popup-icons/favicon.png" alt="user">
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+</body>
+
+
+
+
+
+
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)